.github/workflows/build-and-deploy.yml

```diff
@@ -124,7 +124,8 @@ jobs:
           cd target/generated-docs/search-engine
           npm install
           node ./converter.js ../website/pages/docs/ ../website/pages/explore/ ../katacoda-scenarios/ ../website/pages/architectures/solutions/ .html
-          cp {index.json,docs-json.json} ../website
+          node ./searchSolutions.js ../website/pages/architectures/solutions/ .html
```

Comment:
I would call the script indexSolutions.js and I would put it into the architectures repo into the scripts folder

Reply:
Maybe it is also a good idea to put it into a separate step.